packages/gestalt/src/PopoverEducational.js

```diff
@@ -29,7 +29,7 @@ type PrimaryActionType = {|
 |};
 
 function PrimaryAction({
-  accessibilityLabel,
+  accessibilityLabel = 'Popover',
```

Comment:
hehe not here!

Reply:
This is the button

Reply:
In line 123 we should see the default. Can you make sure you rebase?

![Screenshot by Dropbox Capture](https://user-images.githubusercontent.com/10593890/234358070-d48949ad-0215-41a3-80e5-4f561637009f.png)

Reply:
Argh, got distracted and ended up changing the wrong thing.
Will rebase and fix.

Reply:
Done :)